search-app/test/cmr/search/test/unit/services/acls/granule_acls.clj

```diff
@@ -91,7 +91,7 @@
         old-mem-cache (get-in context [:system :caches :collections-for-gran-acls])
         context (assoc-in context [:system :caches :collections-for-gran-acls] rcache)
         data (.get-value old-mem-cache :collections)]
-    (.set-value rcache :collections-for-gran-acls data)
+    (.set-values rcache :collections-for-gran-acls data)
```

Comment:
Is .set-values a short cut way of writing hash-cache/set-values ?

Reply:
I was trying to keep the code as similar as possible. I can switch over to the other syntax.

Reply:
you don't need to change over, I was just wondering.